generators/server/templates/src/main/java/package/config/_LoggingConfiguration.java

```diff
@@ -84,16 +88,22 @@ public void addLogstashAppender(LoggerContext context) {
         String customFields = "{\"app_name\":\"" + appName + "\",\"app_port\":\"" + serverPort + "\"}";
         <%_ } _%>
 
+        //More documentation is available at: https://github.com/logstash/logstash-logback-encoder
+        LogstashEncoder logstashEncoder=new LogstashEncoder();
```

Comment:
Please be careful of the coding style, put a space after `//`, correctly space arount `=`.